glide.lock

```diff
@@ -25,6 +25,8 @@ imports:
   version: e02fc20de94c78484cd5ffb007f8af96be030a45
 - name: github.com/xeipuuv/gojsonschema
   version: d5336c75940ef31c9ceeb0ae64cf92944bccb4ee
+- name: github.com/Sirupsen/logrus
+  version: 3ec0642a7fb6488f65b06f9040adc67e3990296a
```

Comment:
logrus cuts tagged releases.  Can we use [0.10.0][1] here?

[1]: https://github.com/Sirupsen/logrus/tree/v0.10.0